src/Microsoft.ML.TensorFlow/TensorflowUtils.cs

```diff
@@ -90,7 +90,24 @@ internal static DataViewSchema GetModelSchema(IExceptionContext ectx, Graph grap
                     DataViewType columnType = new VectorDataViewType(mlType);
                     if (!(Utils.Size(tensorShape) == 1 && tensorShape[0] <= 0) &&
                         (Utils.Size(tensorShape) > 0 && tensorShape.Skip(1).All(x => x > 0)))
-                        columnType = new VectorDataViewType(mlType, tensorShape[0] > 0 ? tensorShape : tensorShape.Skip(1).ToArray());
+                        // When treatOutputAsBatched is true we keep the existing behaviour. This means that if the first dimension is greater
```

Comment:
What does “keep the existing behavior” mean in this comment? When someone reads this in a year, what does “existing behavior” mean?

Reply:
I was trying to explain what the existing behavior is with the remainder of the comment. Do you think that I should just remove that line?

Reply:
I think my issue is with "the existing behavior" clause. Once you check in this code, both cases are now "existing behavior".

So maybe just re-word it to
```suggestion
                        // treatOutputAsBatched == true means that if the first dimension is greater
```

Reply:
That makes sense. Actually, I think you have made a similar comment on a prior PR that I did that "a year from now" no one will know what the "existing behavior" is. I'll try and remember that for the future as well.